hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -388,8 +389,8 @@ public void initialize(URI name, Configuration originalConf)
             getMetadataStore(), allowAuthoritative);
       }
       initMultipartUploads(conf);
-      long authDirTtl = conf.getLong(METADATASTORE_AUTHORITATIVE_DIR_TTL,
-          DEFAULT_METADATASTORE_AUTHORITATIVE_DIR_TTL);
+      long authDirTtl = conf.getLong(METADATASTORE_METADATA_TTL,
+          DEFAULT_METADATASTORE_METADATA_TTL);
       ttlTimeProvider = new S3Guard.TtlTimeProvider(authDirTtl);
```

Comment:
do we need to create one if there is no metastore?

Reply:
good find